tests/Agent/IntegrationTests/ContainerApplications/KafkaTestApp/Dockerfile

```diff
@@ -0,0 +1,57 @@
+ARG DISTRO_TAG
+FROM --platform=amd64 mcr.microsoft.com/dotnet/aspnet:7.0-bullseye-slim AS base
```

Comment:
I think you should be able to use `mcr.microsoft.com/dotnet/aspnet:7.0-bullseye-slim` and not need to do all the additional work to install ASPNetCore on lines 5-12.